libs/designsystem/src/lib/components/checkbox/checkbox.component.spec.ts

```diff
@@ -1,89 +1,70 @@
-import { Spectator, createComponentFactory } from '@ngneat/spectator';
-import { IonCheckbox } from '@ionic/angular';
+import { Spectator, createComponentFactory, createHostFactory } from '@ngneat/spectator';
+import { IonCheckbox, IonicModule } from '@ionic/angular';
 
 import { MockComponent } from 'ng-mocks';
 import * as ionic from '@ionic/angular';
-import { SimpleChange } from '@angular/core';
 
+import { DesignTokenHelper } from '../../helpers/design-token-helper';
 import { CheckboxComponent } from './checkbox.component';
 
-describe('CheckboxComponent', () => {
-  const checked = true;
-  const shape = 'square';
-  const color = 'primary';
+const getColor = DesignTokenHelper.getColor;
 
+fdescribe('CheckboxComponent', () => {
   let spectator: Spectator<CheckboxComponent>;
+  let element: HTMLButtonElement;
 
-  const createHost = createComponentFactory({
+  const createHost = createHostFactory({
     component: CheckboxComponent,
-    declarations: [MockComponent(ionic.IonCheckbox)],
+    declarations: [IonCheckbox],
+    imports: [IonicModule.forRoot({ mode: 'ios', _testing: true })],
   });
 
   beforeEach(() => {
-    spectator = createHost({ props: { checked, shape, color } });
+    spectator = createHost(`<kirby-checkbox></kirby-checkbox>`);
+    element = spectator.query('ion-checkbox') as HTMLButtonElement;
   });
 
   it('should create', () => {
     expect(spectator.component).toBeTruthy();
   });
 
   describe('event: checkedChange', () => {
-    it('should emit true', () => {
-      spyOn(spectator.component.checkedChange, 'emit');
-      const change = true;
+    it('should emit true when checked', () => {
+      const checked = true;
 
-      spectator.component.onChecked(change);
+      spyOn(spectator.component.checkedChange, 'emit');
+      spectator.component.onChecked(checked);
 
       expect(spectator.component.checkedChange.emit).toHaveBeenCalledTimes(1);
       expect(spectator.component.checkedChange.emit).toHaveBeenCalledWith(true);
     });
 
-    it('should emit false', () => {
-      spyOn(spectator.component.checkedChange, 'emit');
-      const change = false;
+    it('should emit false when unchecked', () => {
+      const checked = false;
 
-      spectator.component.onChecked(change);
+      spyOn(spectator.component.checkedChange, 'emit');
+      spectator.component.onChecked(checked);
 
       expect(spectator.component.checkedChange.emit).toHaveBeenCalledTimes(1);
       expect(spectator.component.checkedChange.emit).toHaveBeenCalledWith(false);
     });
   });
 
-  describe('Inputs', () => {
-    it('should bind the class .square to ion-checkbox', () => {
-      const squareShape = 'square';
-      spectator.component.ngOnChanges({
-        shape: new SimpleChange(null, squareShape, false),
-      });
+  describe('Input', () => {
```

Comment:
Consider adding a nested `describe('by default')` block that asserts on defaults (when inputs are not set, i.e. `disabled`, `type` etc.)